### PR TITLE
Context Card

### PR DESCRIPTION
**Pull request by ckopecky**
First pass of context card -- might need some tweaking later as we put it with other components.

### CODE REVIEW

File: stories/2-Card.stories.js [hunk 1]
```diff
@@ -130,3 +132,15 @@ export const NoteCard = () => (
     </FlexCenter>
   </Card>
 );
+
+export const ContextCard = () => (
+  <ContextStyle style={{minWidth: "55%", width: "55%"}}>
+    <Flex>
+      <div id="context-div">
```
Comment (bdougie) [inline stories/2-Card.stories.js]: Can you switch this id to a className?

File: src/styles/Card.js [hunk 2]
```diff
@@ -62,4 +62,22 @@ const CardPadding = styled.div`
   padding: ${size.small} ${size.tiny} 0;
 `;
 
-export {Card, CardPadding, FittedCard};
+const ContextStyle = styled(Card)`
+  #context-div {
```
Comment (bdougie) [inline src/styles/Card.js]: This can be a classname. I have a preference for using id's for JavaScript manipulations.

File: src/styles/Card.js [hunk 3]
```diff
@@ -8,6 +8,7 @@ const Card = styled.div`
   border: 0 solid rgba(123, 123, 123, 0.498039);
   border-radius: ${borderRadius};
   font-size: ${size.small};
+  font-family: "Roboto";
```
Comment (bdougie) [inline src/styles/Card.js]: This existing at a higher level: https://github.com/bdougie/open-sauced/blob/master/src/index.css#L84

**Comment by bdougie** [comment]
@ckopecky thanks for opening the PR. I made some comments, everything else looks goods.

Can you do the following: 

1. request a review in the pull request when you are ready for it to be merged. 
2. Sign up for the [Actions beta](https://github.com/features/actions) beta so the tests can run.
3. Add screenshots to the PR so I can review without cloning and running locally.

File: stories/2-Card.stories.js (hunk 1 above)
Comment (ckopecky) [reply stories/2-Card.stories.js]: Done!

**Comment by ckopecky** [comment]
I signed up for Actions Beta and made sure it has permission to run third-party checks.  Anything else I need to do to have it run those tests? @bdougie

**Comment by bdougie** [comment]
> I signed up for Actions Beta and made sure it has permission to run third-party checks. Anything else I need to do to have it run those tests? @bdougie

I have fixed this in this PR: https://github.com/bdougie/open-sauced/pull/364

If you re-run the checks and send a new push it should work.

**Comment by ckopecky** [comment]
I'll go get laptop and run a git pull. Let you know when it's done

**Comment by ckopecky** [comment]
@bdougie ran tests and then pushed after I updated from a `git pull` -- I'm still not seeing it perform all the checks.

**Comment by ckopecky** [comment]
I was looking at the `storybook.yml` file....there's a condition that looks for your repo specifically before running the check. If it's coming from my fork, will it still do the action?  @bdougie

**Comment by bdougie** [comment]
It’s fine the these aren’t affecting the tests anyway it’s just blocking me from on my phone. I will merge soon.

**Pull request merged**